### PR TITLE
add more description to uninstall command during upgrade

### PR DESCRIPTION
**Pull request by richardwxn**

### CODE REVIEW

File: content/en/docs/setup/upgrade/index.md [hunk 1]
```diff
@@ -94,12 +94,20 @@ The output confirms that the pod is using `istiod-canary` revision of the contro
 
 ### Uninstall old control plane
 
-After upgrading both the control plane and data plane, you can uninstall the old control plane. For example, the following command uninstalls a control plane of revision `istio-1-6-5`:
+After upgrading both the control plane and data plane, you can uninstall the old control plane. 
+
+To uninstalls a control plane of revision `istio-1-6-5`, run:
```
Comment (frankbu) [inline content/en/docs/setup/upgrade/index.md:99]:
```suggestion
To uninstall a control plane of revision `istio-1-6-5`, run:
```

File: content/en/docs/setup/upgrade/index.md [hunk 2]
```diff
@@ -94,12 +94,20 @@ The output confirms that the pod is using `istiod-canary` revision of the contro
...
 
 {{< text bash >}}
 $ istioctl x uninstall --revision istio-1-6-5
 {{< /text >}}
 
+But if the existing control plane does not have revision label, you can uninstall it with original installation options, for example:
```
Comment (frankbu) [inline content/en/docs/setup/upgrade/index.md:105]:
```suggestion
If the old control plane does not have a revision label, uninstall it using its original installation options, for example:
```

**Comment by linsun** [comment]
note the cmd doesn't work for me for the default revision, see https://github.com/istio/istio.io/issues/7927

File: content/en/docs/setup/upgrade/index.md (hunk 1 above)
Comment (richardwxn) [reply content/en/docs/setup/upgrade/index.md:99]: done

File: content/en/docs/setup/upgrade/index.md (hunk 2 above)
Comment (richardwxn) [reply content/en/docs/setup/upgrade/index.md:105]: done

**Comment by linsun** [comment]
I don't think this should be merged.  Using the cmd is too dangerous, it would remove istio ingress gw as well.  @richardwxn @frankbu 

```
$ istioctl x uninstall -f manifests/profiles/default.yaml
There are still 5 proxies pointing to the control plane revision :
sleep-579db58c8c-fwrmf.default
helloworld-v1-7d568fc748-7gd5s.default
helloworld-v2-5c55cbd644-n7j4n.default
httpbin-5b87f575f4-tmf72.default
istio-ingressgateway-7bf8767888-9cchz.istio-system
If you proceed with the uninstall, these proxies will become detached from any control plane and will not function correctly. Proceed? (y/N) y
  Removed HorizontalPodAutoscaler:istio-system:istiod.
  Removed ConfigMap:istio-system:istio.
  Removed Deployment:istio-system:istiod.
  Removed ConfigMap:istio-system:istio-sidecar-injector.
  Removed MutatingWebhookConfiguration::istio-sidecar-injector.
  Removed PodDisruptionBudget:istio-system:istiod.
  Removed Service:istio-system:istiod.
  Removed EnvoyFilter:istio-system:metadata-exchange-1.6.
  Removed EnvoyFilter:istio-system:tcp-metadata-exchange-1.6.
  Removed EnvoyFilter:istio-system:stats-filter-1.6.
  Removed EnvoyFilter:istio-system:tcp-stats-filter-1.6.
  Removed EnvoyFilter:istio-system:metadata-exchange-1.7.
  Removed EnvoyFilter:istio-system:tcp-metadata-exchange-1.7.
  Removed EnvoyFilter:istio-system:stats-filter-1.7.
  Removed EnvoyFilter:istio-system:tcp-stats-filter-1.7.
  Removed HorizontalPodAutoscaler:istio-system:istio-ingressgateway.
  Removed Deployment:istio-system:istio-ingressgateway.
  Removed PodDisruptionBudget:istio-system:istio-ingressgateway.
  Removed Role:istio-system:istio-ingressgateway-sds.
  Removed RoleBinding:istio-system:istio-ingressgateway-sds.
  Removed Service:istio-system:istio-ingressgateway.
  Removed ServiceAccount:istio-system:istio-ingressgateway-service-account.
✔ Uninstall complete 
```

**Pull request merged**